utils/zerofrom/derive/src/lib.rs

```diff
@@ -121,12 +121,30 @@ fn zf_derive_impl(input: &DeriveInput) -> TokenStream2 {
     // a newly introduced mirror type parameter that we are borrowing from, similar to C in the original trait.
     // For convenience, we are calling these "C types"
     let generics_env: HashMap<Ident, Option<Ident>> = tybounds
-        .iter()
+        .iter_mut()
         .map(|param| {
             // First one doesn't work yet https://github.com/rust-lang/rust/issues/114393
             let maybe_new_param = if has_attr(&param.attrs, "may_borrow")
                 || may_borrow_attrs.contains(&param.ident)
             {
+                // Remove `?Sized`` bound because we need a param to be Sized in order to take a ZeroFrom of it
```

Comment:
issue: same comment as on Yokeable, I don't agree with this.

Reply:
Ah, this is only on `may_borrow`. This is fine, but please update the docs to say "we need a `may_borrow` param to be `Sized` in order to take a ZeroFrom attribute of it.